### PR TITLE
docs(docs-infra): Drop standalone: true

### PR DESCRIPTION
**Pull request by kirjs**
This is now default value, so safe to remove

### CODE REVIEW

**Review by JeanMeche — APPROVED** [review]
LGTM, thanks for this !

**Comment by AndrewKushnir** [comment]
@kirjs it looks like there was a different GitHub account used to create a commit and create a PR. Could you please update the commit to use the same one?

File: adev/src/content/api-examples/router/testing/test/router_testing_harness_examples.spec.ts [hunk 1]
```diff
@@ -0,0 +1,94 @@
+/**
```
Comment (AndrewKushnir) [inline adev/src/content/api-examples/router/testing/test/router_testing_harness_examples.spec.ts:1]: Do we need this file or it got retained as a rebase artifact?

**Review by AndrewKushnir — APPROVED** [review]
👍

**Comment by pkozlowski-opensource** [comment]
This PR was merged into the repository by commit 6ff53b7437b3edc038800b25bd29387293fe0339.

The changes were merged into the following branches: main, 19.0.x

**Pull request closed without merging**